gateway-provider-security-jwt/src/main/java/org/apache/knox/gateway/provider/federation/jwt/filter/SignatureVerificationCache.java

```diff
@@ -30,8 +30,8 @@
  */
 public class SignatureVerificationCache {
 
-    public static final String JWT_VERIFIED_CACHE_MAX = "jwt.verified.cache.max";
-    public static final int    JWT_VERIFIED_CACHE_MAX_DEFAULT = 250;
+    public static final String VERIFIED_TOKENS_CACHE_MAX = "verified.tokens.cache.max";
```

Comment:
nit: `tokens.verified.cache.max`

Reply:
I'm not sure I understand the nuance between the one I used and your recommendation. Do you think I can ship this way or you insist on changing it?

Reply:
Other similar properties begin with the context (e.g., jwt, sso, etc...), so it seems appropriate to begin with the context (i.e., tokens) with this one also.

Reply:
I've approved the change, so not insistent on changing it, but consistency is good IMO.

Reply:
Ack; I've submitted a new PS.